subsys/net/l2/ieee802154/ieee802154_radio_aloha.c

```diff
@@ -16,28 +18,38 @@ LOG_MODULE_REGISTER(net_ieee802154_aloha, CONFIG_NET_L2_IEEE802154_LOG_LEVEL);
 #include "ieee802154_utils.h"
 #include "ieee802154_radio_utils.h"
 
-static inline int aloha_radio_send(struct net_if *iface,
-				   struct net_pkt *pkt,
-				   struct net_buf *frag)
+static inline int aloha_radio_send(struct net_if *iface, struct net_pkt *pkt, struct net_buf *buf)
```

Comment:
Do you meant the whitespace? If so: This is Zephyr's default clang-format style which I use w/o any exception for lines that I change (and only those) - exactly to avoid that we have to discuss personal preferences.

Do you mean frag -> buf? If so there is a point. Actually what this distinguishes from other places is that a single fragment is to be sent rather than a full buffer. Fixed.